internal/configs/ingress.go

```diff
@@ -230,7 +232,9 @@ func generateNginxCfg(ingEx *IngressEx, apResources AppProtectResources, isMinio
 
 			if cfgParams.HealthCheckEnabled {
 				if hc, exists := ingEx.HealthChecks[ingEx.Ingress.Spec.Backend.ServiceName+ingEx.Ingress.Spec.Backend.ServicePort.String()]; exists {
```

Comment:
How glanceability could be improved here? How about using a shorter key?

Reply:
service and port can identify a upstream